src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -436,12 +436,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                     ->info('validation configuration')
                     ->canBeEnabled()
                     ->children()
-                        ->scalarNode('cache')
-                            ->beforeNormalization()
-                                // Can be removed in 3.0, once ApcCache support is dropped
-                                ->ifString()->then(function ($v) { return 'apc' === $v ? 'validator.mapping.cache.apc' : $v; })
```

Comment:
The previous deprecated notice says, the class ApcCache will be removed, not that the alias "APC" will be discontinued.
This is a new BC break